main.go

```diff
@@ -92,5 +93,10 @@ func Cmds() []cli.Command {
 			Usage:       "Block-level storage, exposed as volumes to mount to host servers. Work with volumes and their associated snapshots.",
 			Subcommands: blockstoragecommands.Get(),
 		},
+		{
+			Name: "orch",
```

Comment:
Let's be explicit here for consistency with the other services: s/orch/orchestration . Tab-completion makes this a non-issue.